ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/DownloadTest.kt

```diff
@@ -34,8 +33,20 @@ class DownloadTest : ClientLoader() {
     @Test
     fun testEchoWithChannelBody() = clientTests {
         test { client ->
-            val size = client.get<ByteReadChannel>("http://www.google.com/").readRemaining().remaining
+            val channel = client.get<ByteReadChannel>("http://www.google.com/")
+            val size = channel.readRemaining().remaining
             assertTrue(size > 0)
         }
     }
+
+    @Test
+    fun testDownload8175() = clientTests {
+        test { client ->
+            repeat(100) {
+                val uri =
+                    "https://raw.githubusercontent.com/ssp/ktor/4f1986df68e3594714ea12949c8af8274be99d01/ktor-client/ktor-client-apache/jvm/test/io/ktor/client/engine/apache/8175.txt"
```

Comment:
We should store this test file in official ktor/ktor repository.

Reply:
Sure. I will replace it with the generated data of specific length.